### ISSUE TITLE
Rails 5: uninitialized constant error

### ISSUE THREAD

**Issue by bdmac** [rails/rails#25840]
### Steps to reproduce

Exception logged in staging environment after upgrading to Rails 5. Unable to provide reproduction steps. Two occurrences logged so far.

### Expected behavior
No exception occurs.

### Actual behavior
Received exception notification reporting:
```
NameError: uninitialized constant ActiveSupport::Multibyte::Chars Did you mean? ActiveSupport::Multibyte::Chars
```
Obviously this is strange for a number of reasons but the primary one being that the DYM output is referencing the exact same constant... as if it actually is loaded/initialized?

The error originated at https://github.com/rails/rails/blob/2efddadd6cba4e2129acedf1d402d11abcc03996/activerecord/lib/active_record/connection_adapters/abstract/quoting.rb#L183 and a full backtrace is available at https://app.honeybadger.io/fault/45385/12a19e6f87c11a2ed803dffe51e6e2ef

While not visible in the public Honeybadger link, the referenced line in our application code that ended up hitting this was simply doing `Model.find(id)` where id was a valid UUID as visible within the ActiveJob/Sidekiq params logged by Honeybadger.

### System configuration
**Rails version**: Rails 5.0.0

**Ruby version**: ruby 2.3.0p0 (2015-12-25 revision 53290) [x86_64-darwin14]

**Comment by rafaelfranca** [comment]
Without a way to reproduce I fear there no much we can do to help with this.

**Comment by bdmac** [comment]
I figured but also didn't want to leave it unreported.

**Comment by solojavier** [comment]
Should we close this issue and reopen if it occurs again or @bdmac finds a way to reproduce it?

**Comment by bdmac** [comment]
Sure I just wanted to make sure it was documented but I can't reproduce it.   
  
  
 ~ Brian McManus  
  
  

  
  
  
  

  
  
>

**Comment by jdurand** [comment]
Like @bdmac I can't reproduce the issue, but I'm logging this just in case it helps someone narrow this down in the future.

I also have a `uninitialized constant ActiveSupport::Multibyte::Chars` exception in my logs and like @bdmac all my code was doing is a simple `Model.find_by_xxx(params[:something])`

One thing that stands out is the User-Agent being `facebookexternalhit/1.1 (+http://www.facebook.com/externalhit_uatext.php)`

I was running Rails v4.2.6 at the time.

**Comment by ajw725** [comment]
just wanted to chime in - just got the same exception. Rails 5.0.1 in API mode with Devise 4.2.0 and devise_token_auth 0.1.39. error seems to have been triggered by Devise's ```current_user``` method, which just runs a db query to find the user by a "uid" field, which is actually an email address. and like the others, i can't reproduce it...just figured i'd add my experience.

**Comment by amatsuda** [comment]
I suppose I just fixed this problem. 8cc828c9b154635ede77acd59a228fd66b0d83d6
The fix will be included in 5.1 and the next 5.0/4.2 releases.
